cli.py

```diff
@@ -38,12 +39,26 @@ def unstar_channel(channel_id):
     r = requests.get(url, params=params)
     r.raise_for_status()
 
+def get_muted_channels():
+    url = BASE_URL + "users.prefs.get"
+    params = {"token": TOKEN}
+    r = requests.get(url, params=params)
+    r.raise_for_status()
+    data = json.loads(r.text)
+    return data["prefs"]["muted_channels"]
+
+def remute_channel(channel_id):
+    url = BASE_URL + "chat.command"
+    params = {"token": TOKEN, "channel": channel_id, "command": "/mute"}
```

Comment:
Is it hidden api?

Reply:
Yep, Marian found that here https://github.com/ErikKalkoken/slackApiDoc